includes/class-article-settings.php

```diff
@@ -103,6 +137,40 @@ public function render_metabox( $post, $args ) {
 		}
 	}
 
+	/**
+	 * Render a custom metabox to check/uncheck whether or not the sharing widget should be hidden
+	 *
+	 * @since 1.0.2
+	 * @param obj $post Post object.
+	 * @param obj $args Arguments object.
+	 */
+	public function render_hide_widget_metabox( $post, $args ){
+
+		$hide_republication_widget = get_post_meta( $post->ID, 'republication-tracker-tool-hide-widget', true );
+
+		$checked = '';
+
+		if( true == $hide_republication_widget ) {
+
+			$checked = 'checked';
+
+		}
+
+		$hide_republication_widget = apply_filters( 'hide_republication_widget', $hide_republication_widget );
+
+		if( true == $hide_republication_widget ){
+			echo '<p>The Republication sharing widget on this post is programatically disabled through the hide_republication_widget filter.</p>';
```

Comment:
The only nitpicks I have here:

- the filter name should be wrapped in a code tag
- this should link to documentation somewhere

Reply:
Updated in https://github.com/INN/republication-tracker-tool/pull/54/commits/8b3782d1090f9a4b8892f7cdd2f75a9156c44150. Note that the link won't work until this PR is merged.

Reply:
Link not working until this PR is merged is fine.

Works with the current branch: https://github.com/INN/republication-tracker-tool/blob/51-toggle-republication-on-off-for-posts/docs/removing-republish-button-from-categories.md